WcaOnRails/app/models/competition.rb

```diff
@@ -118,7 +118,7 @@ def confirmed_or_visible?
   # validation, so people cannot pass a non-delegate as a delegate (even for an old comp).
   # See https://github.com/thewca/worldcubeassociation.org/issues/185#issuecomment-168402252
   # Once that is done, we'll be able to change this validation to work on old competitions.
-  validate :delegates_must_be_delegates
+  validate :delegates_must_be_delegates, unless: :is_over?
```

Comment:
Hmm... I think I'm not wild about the world "valid" here, because it has a specific meaning in Rails model-land. I'm not saying I love `delegates_must_be_delegates`, but I think I prefer uninformative to possibly misleading.

What about `delegates_must_actually_be_delegates`?